pkg/network/master/egressip.go

```diff
@@ -312,12 +312,10 @@ func (eim *egressIPManager) ClaimEgressIP(vnid uint32, egressIP, nodeIP, sdnIP s
 				Node: nodeName,
 			},
 		}
-		if _, err := eim.cloudNetworkClient.CloudV1().CloudPrivateIPConfigs().Create(context.TODO(), &cloudPrivateIPConfig, metav1.CreateOptions{}); err != nil {
-			if kerrors.IsAlreadyExists(err) {
-				if cloudPrivateIPConfig.Spec.Node != nodeName {
-					klog.Infof("CloudPrivateIPConfig: %s is being moved and still exists, enqueuing its creation", egressIP)
-					eim.cloudPrivateIPConfigCreationQueue[egressIP] = cloudPrivateIPConfig
-				}
+		if existingCloudPrivateIPConfig, err := eim.cloudNetworkClient.CloudV1().CloudPrivateIPConfigs().Create(context.TODO(), &cloudPrivateIPConfig, metav1.CreateOptions{}); err != nil {
+			if kerrors.IsAlreadyExists(err) && existingCloudPrivateIPConfig.Spec.Node != nodeName {
```

Comment:
Could you put these conditions in separate blocks, as it was before? The reason I am asking is to not log spurious error messages on L320, which would happen if the object is existing and assigned to the node we want it to.

Reply:
Agreed off-band to keep things as they are